styles/newsletter-dark.scss

```diff
@@ -0,0 +1,32 @@
+.n-newsletter-signup--dark {
+
+    .n-newsletter-signup__header {
+        background: none;
+        border: 1px solid getColor('white-60');
```

Comment:
woudl prefer to define border top and bottom explicitly and not touch border left and right at all

Reply:
actually, I think a single `border-color: getColor('white-60');` woudl do the job, no?